azkeys/coseSigner.go

```diff
@@ -64,6 +64,7 @@ func (f *KeyVaultCoseSignerFactory) NewIdentifiableCoseSigner(ctx context.Contex
 	locationHashString := hex.EncodeToString(locationHash)[:6]
 
 	kv := KeyVaultCoseSigner{
+		ctx:     ctx,
```

Comment:
Putting a context in a struct is dabgerous if it has cancellation. For safety 

     ctx: context.WithoutCancel(ctx),

to remove that possibility

Reply:
done